docs/guide/extensibility/commands.md

```diff
@@ -30,6 +30,15 @@ view.run_command('insert_snippet', {"contents": "<$SELECTION>"})
 view.window().run_command("prompt_select_project")
 ```
 
+## From command line (CLI)
+
+Commands may be passed to Sublime Text directly from the command line
+of the operating system. Usage:
+
+```bash
+subl --command "show_overlay {\"overlay\": \"command_palette\", \"text\":\"hello\"}"

```

Comment:
note that it's generally easier to avoid having to escape all the JSON quotes by using single quotes for the parameter to `--command`:
```sh
subl --command 'show_overlay {"overlay": "command_palette", "text": "hello"}'
```

Reply:
I think I was so glad I found how args are passed that I missed that obvious thing, thanks.